qualtrics_link/util.py

```diff
@@ -226,10 +226,13 @@ def filter_person_list(person_list):
             employee_list.append(person)
 
     # If more than one Person with Employee role, check if any have prime indicator set to 'Y'
+    # Else return the first employee record
     if len(employee_list) > 0:
```

Comment:
Walked thru all test cases. all set. my bad, I was reading line 230 to be >1 not >0.